src/extension/wps/wps-core/src/test/java/org/geoserver/wps/other/NoArgWPS.java

```diff
@@ -0,0 +1,16 @@
+package org.geoserver.wps.other;
```

Comment:
This file should include a copyright header, along the lines of:

```
 /* (c) 2015 Open Source Geospatial Foundation - all rights reserved
  * This code is licensed under the GPL 2.0 license, available at the root
  * application directory.
  */
```
See the developers guide for more info: http://docs.geoserver.org/stable/en/developer/policies/committing.html#commit-guidelines